conans/client/rest/rest_client.py

```diff
@@ -79,32 +79,58 @@ def __init__(self, output, requester, put_headers=None):
         # Remote manager will set it to True or False dynamically depending on the remote
         self.verify_ssl = True
         self._put_headers = put_headers
+        self._server_capabilities = {}  # Dict of url: list
+
+    def store_capabilities(self, response):
```

Comment:
These three functions are to cache the remote capabilities and avoid extra calls.